### PR TITLE
Project with webpack

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Project set us using webpack
- Dynamically created todo list items from a sample array of objects
- Followed minimalistic styling guide

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @abel-tefera,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project in which the readme doc does not list any future feature. Kindly list at minimum one future feature.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    title: 'Eat',
+    description: 'You have to eat to live?',
+    completed: false,
+  },
+  {
+    _id: 3,
+    title: 'Sleep',
+    description: 'You have to sleep to live?',
+    completed: false,
+  },
```
Comment (zieeco) [inline src/index.js]: - Hey, according to the project [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), the object should only contain `index, description, and completed`. please, update your object to have the above properties and remove properties not listed in the requirement.

```
 {
    index: 1,
    description: 'An example of your description',
    completed: false,
  }
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+const main = () => {
+  const todoList = document.querySelector('.todo-list');
+  for (const todo of todos) {
+    const listItem = document.createElement('li');
+    listItem.classList.add('todo-item');
+    const check = document.createElement('input');
+    check.type = 'checkbox';
+    check.classList.add('todo-checkbox');
+    listItem.innerHTML = `${todo.title}`;
```
Comment (zieeco) [inline src/index.js]: - In here, after updating the objects, you shouldn't have the `title` in the object again. Also, from the objects in your array, you're only using the `title`, what about the rest properties in the object? let's improve on this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @abel-tefera,






Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Fair design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
*Please, kindly improve your design*


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @abel-tefera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- Webpack correctly use to bundle the app :+1: 


## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**